CHANGELOG.md

```diff
@@ -4,6 +4,30 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.16.0
+
+This release adds the `withSentryConfig` feature to the Svelte SDK. It replaces the now deprecated Svelte `componentTrackingPreprocessor` which will be removed in the next major release.
+
+- feat(node): Export Span type from `@sentry/types` (#5982)
+- feat(svelte): Add `withSentryConfig` function to wrap User Svelte Configuration (#5936)
+- fix(nextjs): Correctly apply auto-instrumentation to pages in `src` folder (#5984)
+- fix(node): Improve logging in express scope tag match test (#5977)
```

Comment:
This is a test only PR, we can exclude it

Reply:
Whoops, good catch, thanks!